src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('.tree');
+
+for (const li of document.querySelectorAll('li')) {
+  const span = document.createElement('span');
+
+  li.prepend(span);
+  span.append(span.nextSibling);
+}
+
+list.addEventListener('click', (e) => {
```

Comment:
```suggestion
list.addEventListener('click', (event) => {
```
It is better not to shorten the names so much

Reply:
<img width="678" alt="Знімок екрана 2022-12-12 о 22 51 34" src="https://user-images.githubusercontent.com/108242299/207152367-fb5501ad-15c4-4356-af01-e6e1c9b4888c.png">
I do like this in previous tasks and have the same problem